tools/gen-ssh.sh

```diff
@@ -2,8 +2,8 @@
 
 # Create SSH key
 SSH_DATA_DIR="$(mktemp -d)"
-SSH_KEY=${SSH_DATA_DIR}/id_rsa
-ssh-keygen -f "${SSH_KEY}" -N "" -q -t rsa
+SSH_KEY=${SSH_DATA_DIR}/ssh_key
+ssh-keygen -f "${SSH_KEY}" -N "" -q -t ecdsa -b 256 -m pem
```

Comment:
edge-ami-9 and edge-ami-cs9 tests are failing due to AWS does not support ECDSA key type. Is it possible to keep using RSA?

Reply:
I replaced the ECDSA key with an RSA-SHA2 key whis is both FIPS and AWS compatible.